src/simularium/VisGeometry/KMeansWorker.ts

```diff
@@ -17,7 +17,3 @@ class KMeansWorker {
 export type KMeansWorkerType = typeof KMeansWorker;
 
 Comlink.expose(KMeansWorker);
-
-// I know of no other way to deal with this, see the documentation for webpack's worker-loader.
-// eslint-disable-next-line @typescript-eslint/no-explicit-any
-export default self as any;
```

Comment:
The above change is a bonus that should have been removed when updating webpack a few weeks ago.